src/bin/wasm-tools/print.rs

```diff
@@ -30,6 +30,10 @@ pub struct Opts {
     /// Print instructions in the folded format.
     #[clap(short, long)]
     fold_instructions: bool,
+
+    /// The string to use when indenting.
+    #[clap(long)]
+    indent_text: Option<String>,
```

Comment:
Do you think it'd be worthwhile to also have `--indent 4` as a shortcut for `--indent-text '    '`?

Reply:
It's a good idea, but maybe someone would like '\t'?